project-resources/INFRASTRUCTURE_MENU.md

```diff
@@ -21,6 +21,9 @@ The OpenJS Foundation can either manage your DNS for you, or delegate to one of
 ## SSL Certificates
 The OpenJS Foundation will purchase 2-year wildcard SSL certs for each project's managed domains as needed.
 
+## Software Signing Certificates
+The OpenJS Foundation can purchase software signing certificates for Apple and Windows (via DigiCert).
```

Comment:
```suggestion
The OpenJS Foundation can purchase software signing certificates for Mac/iOS and Windows (via DigiCert).
```
(Apple and Microsoft are companies, Mac, iOS, and Windows are operating systems)